app/models/concerns/tipline_content_video.rb

```diff
@@ -3,13 +3,18 @@
 module TiplineContentVideo
   extend ActiveSupport::Concern
 
-  # MP4 less than 10 MB (WhatsApp supports 16 MB, let's be safe)
-  def header_file_video_max_size
+  # Max size that WhatsApp supports
+  def header_file_video_max_size_whatsapp
+    16
```

Comment:
It's better to read the value from config not hard coded

Reply:
Seconded!

Reply:
Done. thanks!